### PR TITLE
Add state subservice option

### PR DESCRIPTION
**Pull request by orbitalturtle**
This is just a little PR that puts the state subservice into place, which we'll need over at lndk. Since the main tonic_lnd repo seems MIA at the moment, Carla suggested I submit this PR to this more active fork

Also, as the README says, is this still pinned to LND v0.15.4-beta? I took the staterpc protos from that version but happy to update.

### CODE REVIEW

**Review by douglaz — APPROVED** [review]
LGTM

**Comment by douglaz** [comment]
@orbitalturtle Thanks for submitting. I'll let others take a look before merging it.

**Review by okjodom — APPROVED** [review]

**Pull request merged**
